langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -141,6 +216,11 @@ public isolated function includes(string str, string substr, int startIndex = 0)
 
 # Tests whether a string starts with another string.
 #
+# ```ballerina
+# string greeting = "Welcome to Ballerina programming language";
+# boolean starts = greeting.startsWith("Welcome");
```

Comment:
Should we use variable names like `startsWithWelcome` or is that too long?

Reply:
Done.